StrategyDesignPattern/Strategy/DriverStrategy.java

```diff
@@ -0,0 +1,3 @@
+public interface DriverStrategy {
+    void drive();
+}
```

Comment:
Let's make this interface protected, otherwise it will spread throughout whole code base.